### DISCUSSION TITLE
Welcome to projoint Discussions!

### DISCUSSION THREAD

**Discussion by yhoriuchi** [yhoriuchi/projoint#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brueckmann** [comment]
Hello! 

As you might already know, I really love your great work on the projoint package and like to use them (again) on some conjoint data of my own. 

Now, I try to understand the error message "Error in pj_estimate(.data = .data, .structure = structure, .estimand = estimand,  : No rows available to estimate IRR/tau." which I received after 

`mm <- projoint(reshaped_data_control, .structure = "profile_level", .estimand = "mm", .irr = 0.656)
`

It is not clear to me, as I supplied the IRR and from the same data projoint can predict tau. Any help appreciated. 

Thank you!

**Comment by yhoriuchi** [comment]
Hi. Thank you for pointing out this issue. I fixed this bug. Please reinstall the package. I believe your code should now work. I also fixed various parts of the code relevant to choosing a specific IRR value. For example, if IRR is provided, there is no need to simulate or bootstrap data to estimate IRR. (We still simulate/bootstrap MMs or AMCEs if we choose the simulate/bootstrap options.) All these relevant issues have been resolved.

**Reply by brueckmann** [reply]
Yes, you are exactly right, now it works, thanks so much!
